sale_automatic_workflow/tests/test_automatic_workflow.py

```diff
@@ -13,33 +13,19 @@
 
 @tagged("post_install", "-at_install")
 class TestAutomaticWorkflow(TestCommon, TestAutomaticWorkflowMixin):
-    def test_full_automatic(self):
+    def test_01_full_automatic(self):
         workflow = self.create_full_automatic()
         sale = self.create_sale_order(workflow)
         sale._onchange_workflow_process_id()
         self.assertEqual(sale.state, "draft")
         self.assertEqual(sale.workflow_process_id, workflow)
         self.run_job()
         self.assertEqual(sale.state, "sale")
-        self.assertTrue(sale.picking_ids)
         self.assertTrue(sale.invoice_ids)
         invoice = sale.invoice_ids
         self.assertEqual(invoice.state, "posted")
-        picking = sale.picking_ids
-        self.run_job()
-        self.assertEqual(picking.state, "done")
-
-    def test_onchange(self):
```

Comment:
I guess we could keep this test on the onchange and rely on the `team_id` field instead of `picking_policy`